### PR TITLE
How to use under ArchLinux from issue#47

### PR DESCRIPTION
**Pull request by Sandman6z**
Thanks to the author's invitation, I made a modification to the environment configuration part of the readme and proposed a PR for issue #47.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +40,13 @@ sudo apt-get update && sudo apt-get install -y build-essential libsdl2-dev cmake
 3. Click the Run and Debug page on the left, and select `Debug LVGL demo with gdb` from the drop-down on the top. Like this:
 ![image](https://github.com/lvgl/lv_port_pc_vscode/assets/7599318/f527b235-5718-4949-b5f0-bd807b3a64ba)
 4. Click the Play button or hit F5 to start debugging.
+   
+* ***ArchLinux User***
```
Comment (kisvegabor) [inline README.md:44]: Simple bold?
```suggestion
**ArchLinux User**  
```

**Review by kisvegabor — COMMENTED** [review]
Thank you! :tada: 

Just a minor formatting note.

**Comment by Sandman6z** [comment]
> Thank you! 🎉
> 
> Just a minor formatting note.

---

Sorry! 😢
This is the first time I have modified the issue, I have modified the suggestion, if you can't submit it successfully, please contact me in time.
Thank you!

**Review by kisvegabor — APPROVED** [review]
Looks good! Thank you! Especially for having your first contribution! 🎉

**Pull request merged**